taipy/core/data/data_node.py

```diff
@@ -415,19 +420,19 @@ def read(self) -> Any:
             )
             return None
 
-    def append(self, data, job_id: Optional[JobId] = None, **kwargs: Dict[str, Any]):
+    def append(self, data, editor_id: Optional[str] = None, **kwargs: Dict[str, Any]):
```

Comment:
kwargs shouldn't be typed like this, more like `**kwargs: Any`
https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs

Reply:
Ah, OK. Thanks! 
I believe that's the type ignore root cause.